rcl_interfaces/msg/ParameterValue.msg

```diff
@@ -6,8 +6,8 @@
 uint8 type
```

Comment:
I've got two thumbs up on my last comment so will consider this question resolved. (in the future, reactions don't send notifications (at least not to me!) so comments would be appreciated)